src/components/QRCodeModal/index.tsx

```diff
@@ -63,7 +63,7 @@ const QRCodeModal: React.FC<QRCodeModalProps> = ({ shortUrl, onClose }) => {
                         src: RDSIcon,
                         height: 50,
                         width: 50,
-                        excavate: true,
+                        excavate: false,
                     }}
                     renderAs="canvas"
                     level="L"
```

Comment:
Still when I run this locally I can't scan 
as QRcode is deprecated and removing image settings makes QR code work 
https://www.npmjs.com/package/qrcode.react#qrcode---deprecated

Reply:
Can you share the QR code image here?

Reply:
Removed RDS logo from QR code

Reply:
After removing logo? I have to send?
and also use QRCodeCanvas instead of QRCode  as QRCode  is deprecated

Reply:
Done